File: java/src/main/java/com/ortecfinance/tasklist/TaskController.java
package com.ortecfinance.tasklist;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import java.util.Arrays;
import java.util.List;

@RestController
@RequestMapping("/tasks")
public class TaskController {

    @GetMapping
    public List<String> getTasks() {
        return Arrays.asList("Task 1", "Task 2", "Task 3");
    }
}

File: kotlin/src/main/kotlin/com/ortecfinance/tasklist/TaskController.kt
package com.ortecfinance.tasklist

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/tasks")
class TaskController {
    @GetMapping
    fun getTasks(): List<String> = mutableListOf("Task 1", "Task 2", "Task 3")
}
